sdks/python/apache_beam/examples/snippets/transforms/aggregation/groupintobatches_test.py

```diff
@@ -31,11 +31,11 @@
 
 def check_batches_with_keys(actual):
   expected = '''[START batches_with_keys]
-[('spring', '🍓'), ('spring', '🥕'), ('spring', '🍆')]
-[('summer', '🥕'), ('summer', '🍅'), ('summer', '🌽')]
-[('spring', '🍅')]
-[('fall', '🥕'), ('fall', '🍅')]
-[('winter', '🍆')]
+('spring', ['🍓', '🥕', '🍆'])
+('summer', ['🥕', '🍅', '🌽'])
+('spring', ['🍅'])
+('fall', ['🥕', '🍅'])
+('winter', ['🍆'])
 [END batches_with_keys]'''.splitlines()[1:-1]
   assert_matches_stdout(
       actual, expected, lambda batch: (batch[0][0], len(batch)))
```

Comment:
This might need to be updated. The third argument to `assert_matches_stdout` expects a function to "normalize" the elements before comparing. This is important for transforms where order is not guaranteed. We might need to do something like:

```py
lambda batch: (batch[0], len(batch[1]))
```

Reply:
Thank you, fixed it.